timescaledb/how-to-guides/compression/manually-compress-chunks.md

```diff
@@ -86,3 +86,24 @@ SELECT compress_chunk(i, if_not_compressed => true)
         now()::timestamp - INTERVAL '3 weeks'
     ) i;
 ```
+
+## Roll up uncompressed chunks into compressed chunks
+
+In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
+a single, compressed, chunk as part of your compression procedure. This allows
+you to have much smaller uncompressed chunks. For example, if you have multiple
```

Comment:
Slightly unclear. What is smaller here is the chunk time interval, not the number of bytes, but this can read either way. Probably good to be more clear on this.